Add 2 AWS Amplify SSR examples one with JS and one with TS

rakannimer opened the pull request:
For easier readability : 

- [Amplify + NextJS JS example](https://github.com/rakannimer/next.js/tree/add-aws-amplify-example/examples/with-aws-amplify)

- [Amplify + NextJS TS example](https://github.com/rakannimer/next.js/tree/add-aws-amplify-example/examples/with-aws-amplify-typescript)

timneutkens commented on examples/with-aws-amplify-typescript/.prettierrc.json:
```diff
@@ -0,0 +1,5 @@
+{
```
Prettierrc is not needed.

rakannimer replied:
Thanks for catching that, removed.

lfades commented:
@rakannimer Can you fix the linter errors 🙏 , you'll have to run `yarn lint-fix` in the root folder

rakannimer commented:
Hey @lfades, fixed the linting issue :+1:

lfades commented:
@rakannimer None of the files added is automatically generated? `graphql/schema.json` looks like something created automatically, if that's the case we should exclude those from the PR

rakannimer commented:
@lfades my bad ! I had added them to make the setup faster but they're not necessary.

I deleted them and updated the READMEs with explanation on how to generate them.
